Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx

```diff
@@ -238,18 +237,18 @@ const MeterEventChannelWindow = (props: IProps) => {
         </div>
         <div className="card-body">
             <div style={{ width: '100%', height: window.innerHeight - 420 }}>
-                <Table<OpenXDA.EventChannel>
+                <ConfigurableTable<OpenXDA.EventChannel>
                     cols={[
                         {
-                            key: 'SourceIndices', field: 'SourceIndices', label: 'Channel', headerStyle: { width: '5%' }, rowStyle: { width: '5%' },
+                            key: 'SourceIndices', field: 'SourceIndices', label: 'Channel', headerStyle: { width: '7%' }, rowStyle: { width: '7%' },
```

Comment:
```suggestion
                            key: 'SourceIndices', field: 'SourceIndices', label: 'Channel', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' },
```

Reply:
The reason I didn't change all the widths to `auto` is that it makes all of the columns equal width no matter the content. If that's what we want, I'll change it for all of them, but I felt it didn't make a lot of sense for many of the fields.

Reply:
Per our discussion marking these as resolved. We will need to revisit column widths separately and @elwills did some testing with these values so they will be ok.